### PR TITLE
Inicial data structure

### PR DESCRIPTION
**Pull request by rafaelnevesalmeida**
The four initial tables schemes and resolvers.

### CODE REVIEW

File: server/.env.default [hunk 1]
```diff
@@ -1,8 +0,0 @@
-# .env
-DB_HOST=db
-POSTGRES_USER=postgres
-POSTGRES_PASSWORD=asdlkj
-POSTGRES_DB=onestack
-MAIL_API=
-WEB_URL=http://localhost:3000
-# WEB_URL=https://onestack-client.herokuapp.com
```
Comment (guidoDutra) [inline server/.env.default]: put file back in and add proper POSTGRES_PASSWORD

File: server/src/dataSources/_migrations/README.md [hunk 2]
```diff
@@ -1,17 +0,0 @@
...
-```
-knex migrate:latest
-```
-
-run `knex --help` for more command and detailed information.
-
-Some references:
-
-https://medium.com/@jaeger.rob/seed-knex-postgresql-database-with-json-data-3677c6e7c9bc
-https://bamtech.gitbook.io/dev-standards/backend/graphql-js/getting-started-with-apollo-server-dataloader-knex.mo
```
Comment (guidoDutra) [inline server/src/dataSources/_migrations/README.md]: Don't remove this file, make editions if necessary

File: server/src/dataSources/_seeds/001_entity_types.js [hunk 3]
```diff
@@ -0,0 +1,18 @@
+exports.seed = async (knex) => {
+  await knex('entity_types').del();
+  await knex.raw('ALTER SEQUENCE entity_types_id_seq RESTART WITH 1');
+  return knex('entity_types').insert([
+    {
+      name: 'Produtor',
```
Comment (guidoDutra) [inline server/src/dataSources/_seeds/001_entity_types.js]: Should we use english names as a standard?

File: server/src/dataSources/_seeds/003_link_types.js [hunk 4]
```diff
@@ -0,0 +1,15 @@
...
+  await knex.raw('ALTER SEQUENCE link_types_id_seq RESTART WITH 1');
+  return knex('link_types').insert([
+    {
+      name: 'Site',
+    },
+    {
+      name: 'Facebook',
+    },
+    {
+      name: 'Instagran',
```
Comment (guidoDutra) [inline server/src/dataSources/_seeds/003_link_types.js]: Instagram

File: server/.env.default (hunk 1 above)
Comment (guidoDutra) [inline server/.env.default]: put file back in and add proper POSTGRES_PASSWORD

File: server/src/dataSources/_seeds/001_entity_types.js (hunk 3 above)
Comment (guidoDutra) [inline server/src/dataSources/_seeds/001_entity_types.js]: Should we use english names as a standard?

File: server/src/dataSources/_seeds/003_link_types.js (hunk 4 above)
Comment (guidoDutra) [inline server/src/dataSources/_seeds/003_link_types.js]: Instagram

File: server/src/dataSources/_migrations/README.md (hunk 2 above)
Comment (guidoDutra) [inline server/src/dataSources/_migrations/README.md]: Don't remove this file, make editions if necessary

File: server/src/dataSources/_migrations/20200306222104_add_link_types_table.js [hunk 5]
```diff
@@ -0,0 +1,10 @@
+exports.up = async knex => knex.schema.createTable(
+  'link_types',
+  (table) => {
+    table.increments('id').primary();
+    table.string('name').notNullable();
+    table.timestamps(false, true);
+  },
+);
+
+exports.down = async knex => knex.schema.dropTable('link_types');
```
Comment (guidoDutra) [inline server/src/dataSources/_migrations/20200306222104_add_link_types_table.js]: All migrations have the same timestamp (20200306222104). Try generating through `knex migrate:make` to generate proper timestamps for each migration

File: server/.env.default (hunk 1 above)
Comment (rafaelnevesalmeida) [reply server/.env.default]: 3ec5a3a

File: server/src/dataSources/_seeds/001_entity_types.js (hunk 3 above)
Comment (rafaelnevesalmeida) [reply server/src/dataSources/_seeds/001_entity_types.js]: 70ea294

File: server/src/dataSources/_seeds/003_link_types.js (hunk 4 above)
Comment (rafaelnevesalmeida) [reply server/src/dataSources/_seeds/003_link_types.js]: 70ea294

File: server/src/dataSources/_migrations/README.md (hunk 2 above)
Comment (rafaelnevesalmeida) [reply server/src/dataSources/_migrations/README.md]: ec9a6fe

File: server/src/dataSources/_migrations/20200306222104_add_link_types_table.js (hunk 5 above)
Comment (rafaelnevesalmeida) [reply server/src/dataSources/_migrations/20200306222104_add_link_types_table.js]: 0acc5d4

File: server/src/dataSources/_seeds/006_entities.js [hunk 6]
```diff
@@ -0,0 +1,32 @@
...
+      city: 'Tangamandápio',
+      region: 'Central',
+      country_code: 'BR',
+      post_code: '12345678',
+      longitude: '-14.197362',
+      latitude: '-47.590000',
+    },
+    {
+      name: 'Florestinha',
+      description: 'farmer',
```
Comment (guidoDutra) [inline server/src/dataSources/_seeds/006_entities.js]: Farmer

Comment (rafaelnevesalmeida) [reply server/src/dataSources/_seeds/006_entities.js]: c5fd853

File: server/src/dataSources/contactItem/create.js [hunk 7]
```diff
@@ -0,0 +1,19 @@
+export default ({ db, tableName }) => async ({
+  contact,
+  contactTypeId,
+  contactId,
+}) => {
+  const result = await db(tableName).insert({
+    contact,
+    contact_type_id: contactTypeId,
```
Comment (guidoDutra) [inline server/src/dataSources/contactItem/create.js:8]: lets consider use camel case on postgres or using [this module](https://www.npmjs.com/package/knex-stringcase) for converting snake case to camel case on knex

File: server/src/dataSources/_migrations/20200318215841_add_countries_table.js.js [hunk 8]
```diff
@@ -0,0 +1,10 @@
+exports.up = async knex => knex.schema.createTable(
```
[the remark this answers is not in the document]
Comment (rafaelnevesalmeida) [reply server/src/dataSources/_migrations/20200318215841_add_countries_table.js.js]: e40dc63

**Pull request merged**